contrib/charts/cert-manager/values.yaml

```diff
@@ -35,3 +38,6 @@ ingressShim:
     # You should only change this if you know what you are doing!
     # tag: v0.2.3
     pullPolicy: Always
+
+# Set these if you need to use custom DNS servers
+customDNSServers: []
```

Comment:
Can be removed and documentation updating to advise on different CLI flags instead.